commec/config/result.py

```diff
@@ -260,28 +261,52 @@ def update_query_flag(self):
             self.benign_status
         }:
             self.screen_status = ScreenStatus.ERROR
+            self.rationale = "There was an error during screening"
+            return
+                
+        if self.benign_status == ScreenStatus.CLEARED_FLAG:
+            self.screen_status = self.benign_status
+            self.rationale = "A sequence of concern was cleared as benign"
             return
 
-        if self.benign_status not in {
-            ScreenStatus.NULL, 
-            ScreenStatus.SKIP
-        }:
+        if self.benign_status == ScreenStatus.CLEARED_WARN:
+            self.screen_status = self.benign_status
+            self.rationale = "A mixed sequence of concern was cleared as benign"
+            return
+
+        if self.benign_status == ScreenStatus.PASS:
             self.screen_status = self.benign_status
+            self.rationale = "No regulated regions of concern"
             return
 
         if self.biorisk_status == ScreenStatus.FLAG:
             self.screen_status = ScreenStatus.FLAG
+            self.rationale = "A known biorisk was detected"
             return
 
-        if (
-            self.protein_taxonomy_status == ScreenStatus.FLAG
-            or self.nucleotide_taxonomy_status == ScreenStatus.FLAG
-        ):
+        if self.protein_taxonomy_status == ScreenStatus.FLAG:
+            self.screen_status = ScreenStatus.FLAG
+            self.rationale = "A regulated protein sequence was detected"
+            return
+
+        if self.nucleotide_taxonomy_status == ScreenStatus.FLAG:
             self.screen_status = ScreenStatus.FLAG
+            self.rationale = "A regulated nucleotide sequence was detected"
             return
 
         if self.biorisk_status == ScreenStatus.WARN:
             self.screen_status = ScreenStatus.WARN
+            self.rationale = "A known virulance factor was detected"
```

Comment:
I think this actually makes it sound more concerning than it is. Maybe:
```suggestion
            self.rationale = "Matched to a virulence factor, but these are often shared between regulated and non-regulated organisms"
```